### DISCUSSION TITLE
new Sonoff Elite device - sonoff THR316d

### DISCUSSION THREAD

**Discussion by philllgood** [arendst/Tasmota#15899] [Ideas]
Hello,
Sonoff has made an evolution on his TH16 device (temperature and humidity), it seems that now it is the Elite device (THR316d for example).
Ant project to add it in the device list? 
Regards
Philippe

**Comment by barbudor** [comment]
Anything in Tasmota happens because someone volunteers for the job, and spend the money for the device...  It can be you 
You can just buy it, open it, find out the GPIO mapping and add it to the template repository
At least the doc say it is based on ESP32 so flashing Tasmota is possible

However, I would see some risk with the support of the builtin LCD. Depending on how it is controlled, support may be easy... or not

**Reply by sfromis** [reply]
Someone is likely to work on the similar Sonoff Pow Elite, and assuming same display (which seems probable), support may be coming already, if feasible. Whether there sensor part will take work to get supported is another question, but as they seem to be able to use the old Sonoff TH sensors, chances should not be too bad.

**Comment by arendst** [comment]
I have both pow and th elite on order. Once received they will likely pop up in Tasmota with ot without LCD support ;-)

**Comment by nobodyman1** [comment]
Look at #15856.

**Reply by philllgood** [reply]
Thanks! I missed it....
